README.md

```diff
@@ -71,7 +71,7 @@ simply hook `window.open` during initialization.  For example:
 ### Preferences
 
 #### <b>config.xml</b>
-- <b>InAppBrowserStatusBarStyle [iOS only]</b>: (string, options 'lightcontent' or 'default'. Defaults to 'default') set text color style for iOS.
+- <b>InAppBrowserStatusBarStyle [iOS only]</b>: (string, options 'lightcontent', 'darkcontent' or 'default'. Defaults to 'default') set text color style for iOS.
```

Comment:
Should probably mention that darkcontent is iOS 13+ only, and a bit more info about the styles could be helpful too

Reply:
I checked Apples API docs and added a description.